src/tests/SharpRaven.UnitTests/Data/HttpRequestBodyConverterTests.cs

```diff
@@ -34,15 +34,40 @@
 using System.IO;
 using System.Text;
 
+using Newtonsoft.Json;
 using NUnit.Framework;
 
 using SharpRaven.Data;
+using System.Collections;
 
 namespace SharpRaven.UnitTests.Data
 {
     [TestFixture]
     public class HttpRequestBodyConverterTests
     {
+        #region TestCaseData
```

Comment:
Please remove it. I believe the associated ReSharper settings should place private classes at the bottom of the parent class, if not, let me know and I'll adjust.